base/commands/serializer/serializer.go

```diff
@@ -0,0 +1,28 @@
+//go:build base
```

Comment:
Could you add the following to all files in this directory?
```suggestion
//go:build std || serializer
```